lib/incorrect_roll_error.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
```

Comment:
Impressive that you're writing errors for this edge case - very diligent of you!